pkgs/applications/networking/peroxide/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
+
+buildGoModule rec {
+  pname = "peroxide";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ljanyst";
+    repo = "peroxide";
+    rev = "v${version}";
+    sha256 = "sha256-qTjpHYJ3e6QmLRvvqtM9IJCO0UFgQRy0Jo7JrtfZsUI=";
+  };
+
+  vendorSha256 = "sha256-tc6bU6V9MEkkIgvW3Sh5fyjTnn5EeFr9Xnjd5pt5H/U=";
+
+  patches = [
+    ./fix-unused-vars.patch
+  ];
+
+  postPatch = ''
+    # These tests connect to the internet, which does not work in sandboxed
+    # builds, so skip these.
+    rm pkg/pmapi/dialer_pinning_test.go \
+       pkg/pmapi/dialer_proxy_provider_test.go \
+       pkg/pmapi/dialer_proxy_test.go
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/ljanyst/peroxide";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ aidalgol ];
+    description = "Unofficial ProtonMail bridge";
+    longDescription = ''
+      Peroxide is a fork of the official ProtonMail bridge that aims to be
+      similar to Hydroxide while reusing as much code from the official bridge
+      as possible.
+    '';
+  };
+
+  passthru.tests.peroxide = nixosTests.peroxide;
```

Comment:
This goes before meta. We should probably correct it from where it was copied, too.

Reply:
I did not copy it from anywhere; I was just unaware of this style convention.